### ISSUE TITLE
Pretrained Group Normalization Models

### ISSUE THREAD

**Issue by mingruimingrui** [pytorch/vision#631]
Hi, would there be any plans to be making pretrained group normalization models?

**Comment by fmassa** [comment]
Yes, thats a good feature request.
We will be adding more models in torchvision soon. I'll open a master issue to keep track of them

**Comment by qiulesun** [comment]
Hi, the mainstream segmentation models, like EncNet (CVPR2018), PSPNet (CVPR2017) and Deeplab series will be added?

**Comment by fmassa** [comment]
@qiulesun we have added pre-trained DeepLabV3 in torchvision with the 0.3 release, together with object detection, instance segmentation and person keypoint detection models
